createdisk.sh

```diff
@@ -11,19 +11,12 @@ OC=${OC:-oc}
 DEVELOPER_USER_PASS='developer:$2y$05$paX6Xc9AiLa6VT7qr2VvB.Qi.GJsaqS80TR3Kb78FEIlIL0YyBuyS'
 
 function get_git_tag {
-    GIT_TAG=$(git describe --exact-match --tags HEAD) || GIT_TAG=
-
-    # Based on code from git-version-gen
-    # Don't declare a version "dirty" merely because a time stamp has changed
-    git update-index --refresh > /dev/null 2>&1
-
-    dirty=`sh -c 'git diff-index --name-only HEAD' 2>/dev/null` || dirty=
-    case "$dirty" in
-        '') ;;
-        *) # Don't build an 'official' version if git tree is dirty
-            GIT_TAG=
-    esac
-    # end of git-version-gen code
```

Comment:
I would not drop this code in this commit, but in a separate one if that's what we want.